tests/validate.test.js

```diff
@@ -277,10 +277,10 @@ describe('validate', () => {
         .then(() => {
           const lib = require('../lib/index');
           const expectedLibEntries = {
-            'module1.js': './module1.js',
-            'module2.js': './module2.js',
-            'handlers/func3/module2.js': './handlers/func3/module2.js',
-            'handlers/module2/func3/module2.js': './handlers/module2/func3/module2.js',
+            'module1': './module1',
```

Comment:
The tests have to be adapted AFTER the changes above, because the used external functions to determine the extension have to be stubbed with sinon.